### PR TITLE
OLS-861: add dev preview banner for OpenShift Lightspeed docs

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s):

Issue: https://issues.redhat.com/browse/OLS-861

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by rh-tokeefe** [comment]
@kalexand-rh or @gaurav-nelson any chance either of you can help me identify the specific problem with this PR? When I looked at the log, I didn't see anything that appears to relate to my change, but I might be overlooking something. 

The release is planned for July 29, so I have a little time to figure this out. Thank you for your time.

File: _templates/_page_openshift.html.erb [hunk 1]
```diff
@@ -67,6 +67,14 @@
       </span>
       <% end %>
 
+      <% if (distro_key == "openshift-lightspeed) %>
```
Comment (kalexand-rh) [inline _templates/_page_openshift.html.erb:70]:
```suggestion
      <% if (distro_key == "openshift-lightspeed") %>
```

Comment (rh-tokeefe) [reply _templates/_page_openshift.html.erb]: thank you @kalexand-rh. I didn't see this file indicated in the logs.  I appreciate your help!

**Comment by kalexand-rh** [comment]
If this is the way you want it to look, it seems fine to me:
image

**Comment by rh-tokeefe** [comment]
@kalexand-rh that content is essentially what was used in the old snippet for dev previews. I know you said you had wanted to make it VERY clear that Lightspeed is a dev preview release, so I used that information. Do you think it's too long to serve as a banner and should be shortened or do you think it's ok?

**Comment by rh-tokeefe** [comment]
@kalexand-rh It seems a bit long to me, but I'm guessing there was a reasonable history as to why the language was in the snippet. If we can shorten it, how about something like this?

OpenShift Lightspeed is a Developer Preview release. Developer Preview features provide early access to upcoming product features in advance of their possible inclusion in a Red Hat product offering, enabling customers to test functionality and provide feedback during the development process. Developer Preview features are not supported by Red Hat in any way and are not functionally complete or production-ready. This documentation is a work in progress and might not be complete or fully tested.

**Comment by kalexand-rh** [comment]
I checked with @ahardin-rh, and even though it's long, it's safest to keep the whole thing.

**Pull request merged**